MatrixKit/Views/MXKReceiptSendersContainer.h

```diff
@@ -31,6 +31,17 @@ typedef NS_ENUM(NSInteger, ReadReceiptsAlignment)
     ReadReceiptAlignmentRight = 1,
 };
 
+/*
+ Protocol to provide interface for actions related to the MXKReceiptSendersContainer view
+ */
+@protocol MXKRecieptSendersContainerDelegate <NSObject>
+
+@optional
+
+- (void)didTapReceiptsContainerWithRestClient:(MXRestClient *)restClient RoomMembers:(NSArray *)roomMembers avatars:(NSArray *)avatars recieptDescriptions:(NSArray *)recieptDescriptions;
+
```

Comment:
You used generic annotations for `ReadReceiptsViewController ` and i think it's a good idea to add them for `MXKRecieptSendersContainerDelegate `. Also **R**oomMembers